color_range_list:
  black:
    max:
    - 86
    - 187
    - 196
    min:
    - 0
    - 30
    - 92
  blue:
    max:
    - 233
    - 150
    - 121
    min:
    - 0
    - 88
    - 0
  green:
    max:
    - 255
    - 121
    - 195
    min:
    - 7
    - 11
    - 15
  red:
    max:
    - 255
    - 255
    - 255
    min:
    - 28
    - 146
    - 98
  tennis:
    max:
    - 255
    - 116
    - 181
    min:
    - 19
    - 79
    - 140
  white:
    max:
    - 255
    - 155
    - 155
    min:
    - 110
    - 100
    - 100
  yellow:
    max:
    - 255
    - 177
    - 184
    min:
    - 195
    - 110
    - 146
